schema_salad/main.py

```diff
@@ -129,6 +129,14 @@ def arg_parser() -> argparse.ArgumentParser:
         help="Optional copyright of the input schema.",
     ),
 
+    parser.add_argument(
+        "--codegen-parser-info",
+        metavar="parser_info",
+        type=str,
+        default=None,
+        help="Optional parser name which is accessible via resulted parser API.",
+    )
+
```

Comment:
Maybe re-use `--codegen-package`?

See also https://github.com/common-workflow-language/schema_salad/blob/5dd637fe792f12c985b5d0056b2494ad3220c107/schema_salad/java_codegen.py#L139

Reply:
IMO it is better to introduce a new option rather than reuse `--codegen-package` option.

The `--codegen-package` option affects the package name of the resulted parser in Java and therefore affects the user code that uses parser API such as `load_document`.
On the other hand, `--codegen-parser-info` that is proposed in this request only affects the return value of `parser_info` function and therefore does not affect any user codes unless they call `parser_info`.

Reply:
can `--codegen-parser-info` have a default value calculated in a similar manner as https://github.com/common-workflow-language/schema_salad/blob/5dd637fe792f12c985b5d0056b2494ad3220c107/schema_salad/java_codegen.py#L139 ; likewise if it is not give, but `--codegen-package` was, it would be nice to default to using that.

Basically: it is great to have this property, and for it to be user specified. I just want it to have the best possible default value available

Reply:
Thank you for the clarification!
I agree with reusing the value of `--codegen-package` for the default value of `--codegen-parser-info`.

Reply:
Done!
Now the value of `--codegen-package` is used for `parser_info` if `--codegen-parser-info` is not provided.
I also added a test for it.